### ISSUE TITLE
special-lispy-raise removes ,@ quote

### ISSUE THREAD

**Issue by Wilfred** [abo-abo/lispy#351]
Given the following s-expression, where `|` is point:

```emacs-lisp
`(foo ,@(bar |(baz)))
```

Pressing `r` (bound to `special-lispy-raise`) gives me the following:

```emacs-lisp
`(foo (baz))
```

whereas I was expecting:

```emacs-lisp
`(foo ,@(baz))
```

(which is the behaviour of `paredit-raise-sexp`)

**Comment by abo-abo** [comment]
This is expected behavior for me. I see `,@` as part of `,@(`, so when that one gets removed by <kbd>r</kbd>, the rest has to go too.

But it might be interesting to have <kbd>r</kbd> produce:

    `(foo ,@|(bar))

And another <kbd>r</kbd> give:

    `(foo |(bar))

**Comment by Wilfred** [comment]
Yep, that was the behaviour I had in mind. I think that would conveniently support both use cases.
